apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditPool/ShowPoolActions.tsx

```diff
@@ -79,7 +76,7 @@ export const ShowPoolActions = () => {
         )}
       </View>
 
-      <ExpandableInfoCard key={id} header={totalFeesTitle} info={feeBreakdown} expanded={isExpanded} />
```

Comment:
Aren't there issues with the target card that it needs to be opened? 🤔

Reply:
I think this applies where there are multiple expandable cards, in this case there is only the selected pool.